### ISSUE TITLE
Make contributors.csv part of phoible/dev ?

### ISSUE THREAD

**Issue by xrotwang** [bambooforest/phoible-scripts#15]
To address issues like https://github.com/cldf-datasets/phoible/issues/34 it would be nice if https://github.com/bambooforest/phoible-scripts/blob/master/to_cldf/contributors.csv was part of `phoible/dev`.

**Comment by bambooforest** [comment]
Sure. We also have a `Contents` field in this file which is towards 

https://github.com/phoible/dev/issues/333

**Comment by xrotwang** [comment]
Yes. Also, I just realized https://github.com/phoible/dev/commit/c1a7e24b3ba43a4f74dceeac72c39b742d80a381 will make the code in `to_cldf` break anyway.

**Comment by xrotwang** [comment]
I'll start rewiring CLDF creation from scratch, then, pulling in `phoible/dev` as submodule, as I have done here https://github.com/tupian-language-resources/tudet/tree/main/raw - git submodules seem to fit really well with the `cldfbench` approach.
